react/Pill/Pill.js

```diff
@@ -9,7 +9,7 @@ import CrossIcon from '../CrossIcon/CrossIcon';
 import ScreenReaderOnly from '../ScreenReaderOnly/ScreenReaderOnly';
 
 type Props = {
-  text: string,
+  text: React$Node,
```

Comment:
This seems confusing to keep this prop as `text` if it's no longer only rendering text.

Is a valid alternative to remove the prop (breaking change) and allow child nodes be rendered instead?

eg:

```
Props = {
  children: ReactNode,
  ...otherProps
}

render: <Pill {...props}>{this.props.children}</Pill>

// usage
<Pill><Text>Some text</Text></Pill>
<Pill><div>Some child node</div></Pill>
```

or swap text for `render` prop:

```
Props = {
  render: ReactNode,
  ...otherProps
}

<Pill render={<Text>Some text</Text>} />
<Pill render={<div>Some child node</div>} />
```

Reply:
I've actually been wanting to make this change anyway—I think you're right that now is a good time to do it.

I've taken the reigns of this PR for now, so I'll add this.

Reply:
To be clear, I'm going to add support for `<Pill>Hello world</Pill>` in addition to `<Pill text="Hello world" />`, to maintain backwards compatibility.

Reply:
Done. Updated the PR title and release notes.